guides/v2.1/coding-standards/technical-guidelines/technical-guidelines.md

```diff
@@ -502,6 +502,44 @@ class SampleEventObserverThatModifiesInputs
 {:start="14.2"}
 14.2. Events used SHOULD be observed as specifically as possible. A `global` subscription to an event SHOULD NOT be used when the area impacted is just `frontend`.
 
+## 15. XML configuration
+15.1 Attribute MUST be used for the configuration values than will not be extended in future by adding child nodes or new attributes.
+15.2 Nodes of the same type that repeat multiple times MUST be wrapped in a grouping node.
+15.3 Unnecessary grouping nodes MUST be avoided.
```

Comment:
Unclear statement. Needs to be more precise. Better to list what should be avoided, or maybe it should be rephrased, maybe "Grouping nodes SHOULD be avoided, except for the same type nodes".
Though this one is under question for me. Need to discuss.